samples/SortBenchmark/qsort/qsort.cpp

```diff
@@ -35,10 +34,10 @@ bool compareb(const void * a, const void * b)
 }
 
 extern "C" __declspec(dllexport)
-void stdqsort(void* buf, int len, int dim)
+void stdquicksort(void* buf, int len, int dim)
 {
 	dim = d;
-	qsort(buf, len, dim, compare);
+	qsort_s(buf, len, dim, compare, null);
```

Comment:
Probably.  It is not actually being actively compiled, so I am not verifying correctness at this point.  
 
Perhaps we can just stash the code in a branch at this point?
 
Date: Wed, 23 Sep 2015 14:36:02 -0700
From: notifications@github.com
To: Prajna@noreply.github.com
CC: sanjeevm@stanfordalumni.org
Subject: Re: [Prajna] Replacing functions & code reorganization: (#53)

In samples/SortBenchmark/qsort/qsort.cpp:

>  {
>  	dim = d;
> -	qsort(buf, len, dim, compare);
> +	qsort_s(buf, len, dim, compare, null);

"extern "C"" makes a function name in C++ have 'C' linkage (thus no name mangling).  But this code is still C++, right?


—
Reply to this email directly or view it on GitHub.